ebusd-2.1.x/en/vaillant/15.uih.csv

```diff
@@ -48,7 +48,9 @@ r;wi,,EhpHeatBufferAvailable,EhpHeatBufferAvailable,,,,"5C00",,,onoff,,,heat buf
 [SW>=250]r,,SysWithCoolingSupport,SysWithCoolingSupport,,,,"9B00",,,onoff,,,
 [SW>=362]r,,reserve1,reserve 1,,,,"9D00",,,UCH,,,for future use
 # ##### HC Names and Tel No #####,,,,,,,,,,,,,
+r;w,,HcName1,aaucHcNames CH1,,,,"4100",,,STR:11,,,name of CH1
```

Comment:
is there really a heating circuit 1? usually this is some internal stuff and I doubt that the name for this is anywhere visible on the controller

Reply:
Hello John, yes there is - see results from the reads of the corresponding calls:
read -c uih HcName1
**HK1**       
read -c uih HcName2
**HK2**       
read -c uih HcName3
**HK3**       
read -c uih HcName4
**HK4**       
read -c uih HcName16
**Speicher**

Reply:
sure it is readable on the bus, but can you see the name of HK1 anywhere in the menu of the controller?

Reply:
if you feel it is more correct, then leave CH1 out, but add CH3